fact_store/fact_store_api.py

```diff
@@ -0,0 +1,85 @@
+import os
+from fact_store.model import EventModel, FeedbackModel, Base
+from pybloom import BloomFilter
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import create_engine
+
+
+class FactStore(object):
+    """
+    FactStore:
+
+    Service for feedback collection on accuracy of machine learning
+    anomaly-detection
+
+    """
+    f = BloomFilter(capacity=1000, error_rate=0.001)
```

Comment:
What happens on the app restart? We start from empty BF? In that case `is_not_anomaly` does not really work, does it?

Reply:
Good catch that is one of the edge cases. at startup I can load the bloom filter with all the feedback predict_ids. Another thing is I don't expect the feedback to be more then 1000.

Reply:
@vpavlin I've opened an issue for this one. #109

https://github.com/AICoE/log-anomaly-detector/issues/109

I'll work on this as a separate PR. I want to keep this PR to just what the title describes and the git issue described associated with this PR.